misc.sk

```diff
@@ -261,12 +196,38 @@ dyes before flattening:
 	(bone[ ]meal|white dye)¦s = minecraft:dye {Damage:15}
 	[any] dye = black dye, red dye, green dye, brown dye, blue dye, purple dye, cyan dye, light gray dye, gray dye, pink dye, lime dye, yellow dye, light blue dye, magenta dye, orange dye, white dye
 
+aquatic update dyes:
+	minecraft version = 1.13 or newer
+	purple dye¦s = minecraft:purple_dye
+	cyan dye¦s = minecraft:cyan_dye
+	light gr(a|e)y dye¦s = minecraft:light_gray_dye
+	gr(a|e)y dye¦s = minecraft:gray_dye
+	pink dye¦s = minecraft:pink_dye
+	(lime|light green) dye¦s = minecraft:lime_dye
+	light blue dye¦s = minecraft:light_blue_dye
+	magenta dye¦s = minecraft:magenta_dye
+	orange dye¦s = minecraft:orange_dye
+
+	minecraft version = 1.13 to 1.13.2
```

Comment:
Have you tested that this works properly? Using two `minecraft version` specifiers in the same tag? Because if it doesn't work properly then some of these aliases will not be defined in either 1.13 or 1.14.

Reply:
are you meaning the stuff within 1.13-1.13.2? No, I didn't even think to try that, I will try it right now and let you know the results

Reply:
reporting back, yes it works a-OK :)

Reply:
shoot, I now realized I screwed up all the "old" dyes in 1.14 (bone meal, cocoa beans) let me fix that up quickly

Reply:
okay, all fixed :)